libraries/transformer/src/main/java/androidx/media3/transformer/EncoderUtil.java

```diff
@@ -150,6 +150,13 @@ public static ImmutableList<Integer> getCodecProfilesForHdrFormat(
           return ImmutableList.of(MediaCodecInfo.CodecProfileLevel.AV1ProfileMain10HDR10);
         }
         break;
+      case MimeTypes.VIDEO_DOLBY_VISION:
+        if (colorTransfer == C.COLOR_TRANSFER_HLG) {
+          return ImmutableList.of(
+              MediaCodecInfo.CodecProfileLevel.DolbyVisionProfileDvheSt);
```

Comment:
Out of all profile, why have we choose this one?

Reply:
Good question. All exist devices which support Dolby Vision encoding on market will encode the input stream to this profile only. In future, maybe we will extend this list. But Dolby has not finalized which profiles we should support for encoding.